.github/workflows/build-pr.yml

```diff
@@ -8,17 +8,17 @@ jobs:
     env:
       GO111MODULE: "on"
     steps:
+      - name: Checkout code into the Go module directory
+        uses: actions/checkout@v2
+
       - name: Set up Go 1.17
```

Comment:
Looks like these should be updated as well (version removed) 😅

Reply:
Oh, but there's a `GO_VERSION` defined in `vars.mk` (which looks to be an old version of Golang); wondering if things need to be synced somehow (perhaps a global env-var that's set in the workflow?): https://github.com/docker/scan-cli-plugin/blob/5819521a544cda624dae3d53dc37f44cc8d2b7f4/vars.mk#L7